### PR TITLE
Update deployment-guide.mdx

### PR DESCRIPTION
**Pull request by gtelnet**
added link to discord post for performance comparison of validator install methods

### CODE REVIEW

File: docs/mine-hnt/validators/mainnet/deployment-guide.mdx [hunk 1]
```diff
@@ -108,7 +108,11 @@ If nothing is returned from this command, your host system does not have AVX sup
...
+You have three options for deploying the Validator. The deb package, a community member maintained package by PaulVMo, is the least complex to deploy and has very high performance. The Source build has the highest level of complexity to deploy, requires a higher level of Linux technical expertise, and also has very high performance. The docker version is less complex to deploy than Source, if you already have experience with Docker, but does not perform as well as the other two options. 
+
+See this [discord post](https://discord.com/channels/404106811252408320/857353417739730954/934259909434945536) on Helium discord for a performance comparison of the three different validator install methods.
```
Comment (madninja) [inline docs/mine-hnt/validators/mainnet/deployment-guide.mdx:113]: Should we inline the description here?

Comment (gtelnet) [reply docs/mine-hnt/validators/mainnet/deployment-guide.mdx:113]: Does that mean change it to this? [Performance comparison of the three different validator install methods](https://discord.com/channels/404106811252408320/857353417739730954/934259909434945536)

Comment (madninja) [reply docs/mine-hnt/validators/mainnet/deployment-guide.mdx:113]: > Does that mean change it to this? [Performance comparison of the three different validator install methods](https://discord.com/channels/404106811252408320/857353417739730954/934259909434945536)

That’s already better, but I meant actually migrating the content from discord here

Comment (gtelnet) [reply docs/mine-hnt/validators/mainnet/deployment-guide.mdx:113]: ok, wasn't sure if we could embed screenshots here, which is the crux of that discord post. will look into it.

**Comment by gtelnet** [comment]
will update/resubmit

**Pull request closed without merging**